lockpick-libzfs-abi.sh

```diff
@@ -0,0 +1,210 @@
+#! /usr/bin/env bash
+
+# Try to "lockpick" the settings relevant for libzfs.jar on this host OS
+# Note that the "correct answer" may change across OS updates... as well
+# as libzfs.jar evolution. Requires sources, "mvn" and JDK at this time,
+# and you may have to allow tests in pom.xml (set "<skip>" to "false").
+# Note it is likely to leave around Java coredump files, or at least logs
+# of those with stack traces.
+#
+# TODO: Add an option to libzfstest to run just a certain native routine.
+#
+# Copyright (C) 2017 by Jim Klimov
```

Comment:
That's one point I was not sure about. I have no issues putting this out as MIT or whatever is deemed fit, but the rest of the repo (wherever mentioned) is under CDDL, so I think it applies better in this case. 

Note also those other source files are using an older heading for that licensing notice (URLs to OpenSolaris.org lead nowhere nowadays). If @kohsuke is ok with a large-sweeping change, the headings can be reworded to a pattern using the copy of CDDL on http://www.illumos.org/license/CDDL or https://opensource.org/licenses/CDDL-1.0 (perhaps with header wording e.g. from templates like https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/templates/configure.mk) - but I think such changes better be committed and signed off by @kohsuke himself to avoid questions in the future.